app/src/main/java/studyBuddy/timemanagement/StrategyRecord.java

```diff
@@ -0,0 +1,42 @@
+package studyBuddy.timemanagement;
+
+import androidx.annotation.NonNull;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import java.util.Date;
+
+/**
+ * Stores all information necessary to represent a session at some point in time
+ */
+public class StrategyRecord {
```

Comment:
Name is confusing - reading it seems like it would implement the strategy interface which would be weird

Reply:
Good point, maybe just calling it "sessionrecord" or just "record" ?